client/cli/src/grpc_client.rs

```diff
@@ -44,7 +42,9 @@ const MAX_GRPC_RETRY_COUNT: u64 = 2;
 pub struct GRPCClient {
     /// The client connection to an AdmissionControl service. We will only connect
     /// when the first request is made.
+    /// TODO deprecate this completely once migration to JSON RPC is complete
     client: AdmissionControlClientBlocking,
+    json_rpc_client: JsonRpcClientBlocking,
```

Comment:
nit: I frankly don't like original name either. Can we just call it JsonRpcClient. That blocking suffix is weird. Or at least JsonRpcBlockingClient. But I'd prefer just JsonRpcClient